static/css/redox.css

```diff
@@ -43,11 +48,11 @@ nav li a {
 }
 
 .topbar {
-  width: 100%;
+  width: 92%;
   position: fixed;
-  top: 0;
-  left: 0;
-  height: 4em;
+  top: 0px;
```

Comment:
`px` is not needed and it is common practice to omit unit when using `0` as a value. I would like to keep that.